lua/caramba/git.lua

```
@@ -519,30 +524,181 @@ M.review_code = function()
+      current_file .. ".spec." .. current_ext,
+      "test_" .. current_file .. "." .. current_ext,
+      current_file .. ".d.ts", -- TypeScript definitions
+    }
+    
+    related_files = "\n## Potentially Related Files:\n"
+    for _, pattern in ipairs(patterns) do
+      local full_path = current_dir .. "/" .. pattern
+      if vim.fn.filereadable(full_path) == 1 then
+        related_files = related_files .. "- " .. pattern .. " (test file)\n"
+        -- Read first few lines to understand test structure
+        local test_lines = vim.fn.readfile(full_path, '', 20)
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)